[2021-10-10 19:31:04,650] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: main.data.scatter_group_data.scatter_data 2021-10-10T16:30:58.729934+00:00 [queued]>
[2021-10-10 19:31:04,656] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: main.data.scatter_group_data.scatter_data 2021-10-10T16:30:58.729934+00:00 [queued]>
[2021-10-10 19:31:04,657] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 19:31:04,657] {taskinstance.py:1096} INFO - Starting attempt 1 of 2
[2021-10-10 19:31:04,657] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 19:31:04,664] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): data.scatter_group_data.scatter_data> on 2021-10-10T16:30:58.729934+00:00
[2021-10-10 19:31:04,665] {standard_task_runner.py:52} INFO - Started process 49765 to run task
[2021-10-10 19:31:04,668] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'main', 'data.scatter_group_data.scatter_data', '2021-10-10T16:30:58.729934+00:00', '--job-id', '674', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmphlz7l18k', '--error-file', '/tmp/tmpqpnez8nz']
[2021-10-10 19:31:04,668] {standard_task_runner.py:77} INFO - Job 674: Subtask data.scatter_group_data.scatter_data
[2021-10-10 19:31:04,686] {logging_mixin.py:109} INFO - Running <TaskInstance: main.data.scatter_group_data.scatter_data 2021-10-10T16:30:58.729934+00:00 [running]> on host abd
[2021-10-10 19:31:04,708] {taskinstance.py:1252} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=main
AIRFLOW_CTX_TASK_ID=data.scatter_group_data.scatter_data
AIRFLOW_CTX_EXECUTION_DATE=2021-10-10T16:30:58.729934+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-10T16:30:58.729934+00:00
[2021-10-10 19:31:04,708] {python.py:151} INFO - Done. Returned value was: None
[2021-10-10 19:31:04,711] {taskinstance.py:1212} INFO - Marking task as SUCCESS. dag_id=main, task_id=data.scatter_group_data.scatter_data, execution_date=20211010T163058, start_date=20211010T163104, end_date=20211010T163104
[2021-10-10 19:31:04,759] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-10 19:31:04,780] {local_task_job.py:261} INFO - 5 downstream tasks scheduled from follow-on schedule check
